### PR TITLE
changes frontpage banner to elife

### PR DESCRIPTION
**Pull request by lenazun**
nada más

### CODE REVIEW

File: h/templates/home.html.jinja2 [hunk 1]
```diff
@@ -223,7 +223,7 @@
           </div>
         </div>
         <div class="press-release-banner">
-            <p>Hypothesis announces a coalition of over 40 scholarly organizations bringing annotation to all knowledge. <a href="https://hypothes.is/annotating-all-knowledge/"><b>Learn more</b></a></p>
+            <p><a href="https://hypothes.is/blog/elife-partners-w…larly-annotation/">eLife Partners with Hypothes.is to Advance Open Scholarly Annotation.</a></p>
```
Comment (nickstenning) [inline h/templates/home.html.jinja2]: Can we use "Hypothesis" (not "Hypothes.is") in prose text, as per our [brand guidelines](https://docs.google.com/document/d/1DE5VVgY-fUYRxh9Dn8bcXzNVP4DSx5CrOlM2755I8Hg/edit).

Comment (nickstenning) [inline h/templates/home.html.jinja2]: Oh, and the URL is truncated here. `elife-partners-w…larly-annotation` should be `elife-partners-with-hypothes-is-to-advance-open-scholarly-annotation`.

Comment (chdorner) [inline h/templates/home.html.jinja2]: I just fixed both of these.

**Comment by nickstenning** [comment]
Thanks, Christof!

**Pull request merged**